behave/runner.py

```diff
@@ -598,6 +649,69 @@ def stop_capture(self):
     def teardown_capture(self):
         self.capture_controller.teardown_capture()
 
+    def run_one_feature(self, feature):
+        try:
+            self.feature = feature
+            for formatter in self.formatters:
+                formatter.uri(feature.filename)
+
+            def read_timeout(tags):
+                tag_name = "timeout="
```

Comment:
This is probably a bit Ad-hoc for our case, but right know I don't have any suggestion.
Maybe opening a PR upstream would help.

Reply:
Of course it is Ad-hoc for our case, in fact I was thinking more a file patch than in an upstream PR. Definitively reading tag here does not follow the original project philosophy.